azkaban-common/src/main/java/azkaban/jobtype/JobTypeManager.java

```diff
@@ -309,6 +309,7 @@ private ClassLoader loadJobTypeClassLoader(File pluginDir,
     }
 
     // each job type can have a different class loader
+    logger.info(String.format("Classpath for plugin[dir: %s, JobType: %s]: %s", pluginDir, jobTypeName, resources));
```

Comment:
resources is a list, can it be printed out in this way? Or we need to join together as a string?